docs/TutorialReact.md

```diff
@@ -271,6 +273,36 @@ test('CheckboxWithLabel changes the text after click', () => {
 
 The code for this example is available at [examples/enzyme](https://github.com/facebook/jest/tree/master/examples/enzyme).
 
+#### react-testing-library
+
+You have to run `yarn add --dev react-testing-library` to use react-testing-library.
+
+Let's rewrite the test from above using react-testing-library instead of Enzyme.
+
+```js
+// __tests__/CheckboxWithLabel-test.js
+import React from "react";
+import { render, fireEvent, cleanup } from "react-testing-library";
+import CheckboxWithLabel from "../CheckboxWithLabel";
+
+// automatically unmount and cleanup DOM after the test is finished.
+afterEach(cleanup);
+
+test("CheckboxWithLabel changes the text after click", () => {
+  const { queryByLabelText, getByLabelText } = render(
+    <CheckboxWithLabel labelOn="On" labelOff="Off" />
+  );
+
+  expect(queryByLabelText("Off")).toBeTruthy();
```

Comment:
I would change this to:

```javascript
expect(getByLabelText(/off/i)).toBeTruthy()
```

Same for the one below. Prefer case-insensitive regex over strings 👍

Reply:
Thanks for pointing that out! I updated both the docs and example code 👍 

I amended my commit, which is not so nice for looking at the diff now but I figured it was small enough that it was ok 😅